docs/openrpc.json

```diff
@@ -689,6 +689,40 @@
             },
             "tags": [{ "name": "alpha API"}]
         },
+        {
+            "name": "eth_getFilterChanges",
+            "summary": "Gets the latest results since the last request for a provided filter according to its type",
```

Comment:
Can we add the alpha label to all the filter APIs for now on this file

Reply:
Thanks for adding it here.
If you can add it on all filter methods that would be great also

Reply:
All of the other methods have that label, it was missing only here.